scripts/generate_report.sh

```
@@ -3,226 +3,394 @@
+# リポジトリ数が多い場合はpaginateが必要な場合もあります
+USER_REPOS_JSON=$(gh api graphql -f query='
+  query($owner: String!) {
+    user(login: $owner) {
+      repositories(first: 100, ownerAffiliations: OWNER, isFork: false) {
+        nodes {
+          name
+        }
+      }
+    }
+  }
+' -F owner="$USERNAME")
```

Shellcheck (0.10.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)